lib/ansible/modules/system/mount.py

```diff
@@ -551,23 +539,21 @@ def get_linux_mounts(module, mntinfo_file="/proc/self/mountinfo"):
 def main():
     module = AnsibleModule(
         argument_spec=dict(
-            boot=dict(default='yes', choices=['yes', 'no']),
-            dump=dict(),
-            fstab=dict(default=None),
-            fstype=dict(),
-            path=dict(required=True, aliases=['name'], type='path'),
-            opts=dict(),
+            boot=dict(type='bool', default=True),
+            dump=dict(type='str'),
+            fstab=dict(type='str'),
+            fstype=dict(type='str'),
+            path=dict(type='path', required=True, aliases=['name']),
+            opts=dict(type='str'),
             passno=dict(type='str'),
             src=dict(type='path'),
-            state=dict(
-                required=True,
-                choices=['present', 'absent', 'mounted', 'unmounted']),
+            state=dict(type='str', required=True, choices=['absent', 'mounted', 'present', 'unmounted']),
         ),
         supports_check_mode=True,
         required_if=(
             ['state', 'mounted', ['src', 'fstype']],
-            ['state', 'present', ['src', 'fstype']]
-        )
+            ['state', 'present', ['src', 'fstype']],
```

Comment:
This is unnecessary change.

Reply:
It's mentioned in PEP8. Check all their examples, they all do it that way and for good reason.
So functionally yes, there's no difference, but that's the case for everything in PEP8.
https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas